src/backend/core/category/views.py

```diff
@@ -90,6 +97,7 @@ class CategoryStorefrontView(APIView):
     Used for storefront.
     """
 
+    @check_user_is_staff_decorator()
```

Comment:
Remove the decorator from here, cause this view is intended for storefront

Reply:
Fixed.